ICSharpCode.ILSpyX/Analyzers/Builtin/TypeExposedByAnalyzer.cs

```diff
@@ -22,12 +22,15 @@
 
 namespace ICSharpCode.ILSpyX.Analyzers.Builtin
 {
+	using System.ComponentModel.Composition;
```

Comment:
This should be moved out of the namespace.

Reply:
fixed